doc/contribute_models.md

```diff
@@ -85,6 +75,8 @@ Note that we defined a class named `MyAwesomeClassifier` which will be used as t
 - `loss`: define the default loss function used when training.
 - `prepare_prediction_column`: define how to process the prediction output.
 
+### Write Models Other Than Keras
```

Comment:
Added explanations in this section:

> Note that the model class still need to extend `tf.keras.Model` so that SQLFlow can recognize it's a custom model and execute the function `sqlflow_train_loop` when training. Please refer to our [Deep Embedding Clustering Model](https://github.com/sql-machine-learning/models/blob/develop/sqlflow_models/deep_embedding_cluster.py) as an example.